### ISSUE TITLE
Poly error. Poly check error. Poly Shell error - it's Poly! Or java

### ISSUE THREAD

**Issue by DVious** [polyfy/polylith#214]
**Describe the bug**
Poly in the command line produces:
#error {
 :cause Don't know how to create ISeq from: clojure.lang.Keyword
 :via
 [{:type java.lang.IllegalArgumentException
   :message Don't know how to create ISeq from: clojure.lang.Keyword
   :at [clojure.lang.RT seqFrom RT.java 557]}]
 :trace
 [[clojure.lang.RT seqFrom RT.java 557]
  [clojure.lang.RT seq RT.java 537]
  [clojure.core$seq__5467 invokeStatic core.clj 139]
  [polylith.clj.core.workspace_clj.namespaces_from_disk$prefix_list__GT_lib_strs invokeStatic namespaces_from_disk.clj 27]
  [polylith.clj.core.workspace_clj.namespaces_from_disk$prefix_list__GT_lib_strs invoke namespaces_from_disk.clj 27]
  [clojure.core$map$fn__5935 invoke core.clj 2772]
  [clojure.lang.LazySeq sval LazySeq.java 42]
  [clojure.lang.LazySeq seq LazySeq.java 58]
  [clojure.lang.RT seq RT.java 535]
  [clojure.core$seq__5467 invokeStatic core.clj 139]
  [clojure.core$map$fn__5935 invoke core.clj 2763]
  [clojure.lang.LazySeq sval LazySeq.java 42]
  [clojure.lang.LazySeq seq LazySeq.java 51]
  [clojure.lang.Cons next Cons.java 39]
  [clojure.lang.RT boundedLength RT.java 1790]
  [clojure.lang.RestFn applyTo RestFn.java 130]
  [clojure.core$apply invokeStatic core.clj 667]
  [clojure.core$mapcat invokeStatic core.clj 2800]
  [clojure.core$tree_seq$walk__6455$fn__6456 invoke core.clj 4970]
  [clojure.lang.LazySeq sval LazySeq.java 42]
  [clojure.lang.LazySeq seq LazySeq.java 51]
  [clojure.lang.LazySeq more LazySeq.java 87]
  [clojure.lang.RT more RT.java 722]
  [clojure.core$rest__5453 invokeStatic core.clj 73]
  [clojure.core$flatten invokeStatic core.clj 7204]
  [polylith.clj.core.workspace_clj.namespaces_from_disk$import invokeStatic namespaces_from_disk.clj 58]
  [polylith.clj.core.workspace_clj.namespaces_from_disk$import invoke namespaces_from_disk.clj 56]
  [clojure.core$map$fn__5935 invoke core.clj 2770]
  [clojure.lang.LazySeq sval LazySeq.java 42]
  [clojure.lang.LazySeq seq LazySeq.java 51]
  [clojure.lang.RT seq RT.java 535]
  [clojure.core$seq__5467 invokeStatic core.clj 139]
  [clojure.core$apply invokeStatic core.clj 662]
  [clojure.core$mapcat invokeStatic core.clj 2800]
  [polylith.clj.core.workspace_clj.namespaces_from_disk$imports invokeStatic namespaces_from_disk.clj 69]
  [polylith.clj.core.workspace_clj.namespaces_from_disk$__GT_namespace invokeStatic namespaces_from_disk.clj 81]
  [polylith.clj.core.workspace_clj.namespaces_from_disk$source_namespaces_from_disk$fn__25462 invoke namespaces_from_disk.clj 89]
  [clojure.core$mapv$fn__8535 invoke core.clj 6979]
  [clojure.lang.PersistentVector reduce PersistentVector.java 343]
  [clojure.core$reduce invokeStatic core.clj 6885]
  [clojure.core$mapv invokeStatic core.clj 6970]
  [polylith.clj.core.workspace_clj.namespaces_from_disk$source_namespaces_from_disk invokeStatic namespaces_from_disk.clj 88]
  [polylith.clj.core.workspace_clj.namespaces_from_disk$namespaces_from_disk invokeStatic namespaces_from_disk.clj 94]
  [polylith.clj.core.workspace_clj.components_from_disk$read_component invokeStatic components_from_disk.clj 11]
  [polylith.clj.core.workspace_clj.components_from_disk$read_components$fn__25636 invoke components_from_disk.clj 34]
  [clojure.core$map$fn__5935 invoke core.clj 2772]
  [clojure.lang.LazySeq sval LazySeq.java 42]
  [clojure.lang.LazySeq seq LazySeq.java 51]
  [clojure.lang.Cons next Cons.java 39]
  [clojure.lang.RT length RT.java 1782]
  [clojure.lang.RT seqToArray RT.java 1723]
  [clojure.lang.LazySeq toArray LazySeq.java 132]
  [clojure.lang.RT toArray RT.java 1696]
  [clojure.core$to_array invokeStatic core.clj 346]
  [clojure.core$sort invokeStatic core.clj 3114]
  [clojure.core$sort_by invokeStatic core.clj 3120]
  [clojure.core$sort_by invokeStatic core.clj 3120]
  [polylith.clj.core.workspace_clj.components_from_disk$read_components invokeStatic components_from_disk.clj 33]
  [polylith.clj.core.workspace_clj.core$toolsdeps_ws_from_disk invokeStatic core.clj 103]
  [polylith.clj.core.workspace_clj.core$workspace_from_disk invokeStatic core.clj 148]
  [polylith.clj.core.workspace_clj.interface$workspace_from_disk invokeStatic interface.clj 4]
  [polylith.clj.core.command.core$read_workspace invokeStatic core.clj 52]
  [polylith.clj.core.command.core$workspace_reader_fn$fn__25747 invoke core.clj 59]
  [polylith.clj.core.command.core$execute invokeStatic core.clj 66]
  [polylith.clj.core.command.interface$execute_command invokeStatic interface.clj 4]
  [polylith.clj.core.poly_cli.core$_main invokeStatic core.clj 31]
  [polylith.clj.core.poly_cli.core$_main doInvoke core.clj 7]
  [clojure.lang.RestFn invoke RestFn.java 397]
  [clojure.lang.AFn applyToHelper AFn.java 152]
  [clojure.lang.RestFn applyTo RestFn.java 132]
  [polylith.clj.core.poly_cli.core main nil -1]]}

The problem arose previously, only spotted when 'poly create cs...' failed.
I can't recall the solution but, it may be to do with java versions, which I upgraded to 17, downgraded to 11, then 8, then back to 17.

**To Reproduce**
1. I have no idea how to reproduce it - it occurs from the getgo.

**Expected behavior**
I would like to run poly from the command line.  Upgrading to the latest poly did not help.

**Screenshots**

**Workspace Attachment**
If applicable, add output of following command as attachment:
- `poly ws out:ws.edn`

**Operating System (please complete the following information):**
 - OS: [e.g. MacOS] - Mojave 
 - Version: [e.g. 10.13.6] 10.14
 - java -version returns: 
 - openjdk version "17.0.1" 2021-10-19
 - OpenJDK Runtime Environment GraalVM CE 21.3.0 (build 17.0.1+12-jvmci-21.3-b05)
 - OpenJDK 64-Bit Server VM GraalVM CE 21.3.0 (build 17.0.1+12-jvmci-21.3-b05, mixed mode, sharing)

**Versions (please complete the following information):**
 - Java: [e.g. 1.8.0_151]
 - Poly: [e.g. 0.1.0] poly -version fails but, it's the very latest (2.14)
 - Brew: [e.g. 2.5.6-112-g0d56c47]

**Additional context**
Polylith was previously working and, this problem previously occurred.

Whether by trial and error, and trying different jdks, the problem was resolved for a while.

I just tried to brew upgrade all - it failed but, running 'poly' afterward worked!
But, now, having closed/reopened a terminal, it doesn't work.

**Comment by DVious** [comment]
I ran polylith from no project and, it works!

The problem DOES appear to be in MY development project so, the error occurs when polylith consumes my project.

Is there a guide to this as, however I am mangling my project, I keep doing it!

For example, if I copy/paste/rename a component, which I often do, will that eventually break the polylith aspect of the project?

**Comment by tengstrand** [comment]
It looks like the `poly` tool has problems reading a namespace. If you can figure out how that namespace looks looks like, then please post it here!

**Comment by DVious** [comment]
We did it!

Although I don't totally get it.  

I did a namespace check and, made them all consistent. A good thing, that didn't resolve the problem.

So, I removed all component folders, to achieve 'OK in  poly check.

I restored each folder until one failed poly check, then, eliminated it.

If it helps it was "keycloak-clojure" and, it might be different in being a docker component, with poor documentation, for my thinking.

My require statements in that namespace caused the issue and, as I intend to use some, non-docker, solution, I left it at that.

All my other folders, including cut/paste folders passed poly check and so, I am restored!

Thanks
